components/automate-cli/pkg/verifyserver/services/softwareversionservice/softwareversionservice.go

```diff
@@ -0,0 +1,184 @@
+package softwareversionservice
+
+import (
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/utils/getosutils"
+	"github.com/chef/automate/lib/logger"
+	"github.com/pkg/errors"
+)
+
+type ISoftwareVersionService interface {
+	GetSoftwareVersionServices(string) (*models.SoftwareVersionDetails, error)
+}
+
+type SoftwareVersionService struct {
+	cmdCheckArray []string
+	osFilepath    string
+	logger        logger.Logger
+}
+
+func NewSoftwareVersionService(logger logger.Logger) ISoftwareVersionService {
+	return &SoftwareVersionService{
+		cmdCheckArray: cmdCheckArray,
+		osFilepath:    OSFILEPATH,
+		logger:        logger,
+	}
+}
+
+const (
+	AVAILABILITY        = " availability"
+	ENSURE              = "Ensure "
+	OSFILEPATH          = "/etc/os-release"
+	LINUX_VERSION_CHECK = "Linux Version Check"
+)
+
+var cmdCheckArray = []string{"mkdir", "useradd", "chown", "rm", "touch", "truncate", "echo", "sleep", "ls", "grep", "yum", "which", "cp", "curl", "bash", "sysctl", "cat", "sed", "mount", "pvcreate", "vgcreate", "lvcreate", "mv", "systemd", "wget", "exec"}
```

Comment:
these are not required 
pvcreate
vgcreate 
lvcreate

Reply:
Done